tests/run-make/sigpipe-in-child-processes/main.rs

```diff
@@ -0,0 +1,33 @@
+// Checks the signal disposition of `SIGPIPE` in child processes. Without any
+// `unix_sigpipe` attribute, `SIG_IGN` is the default. But there is a
+// difference in how `SIGPIPE` is treated in child processes with and without
+// the attribute. Search for `unix_sigpipe_attr_specified()` in the code base to
+// learn more.
+
+// Note that tests for the attribute that does not involve child processes are
+// in tests/ui/attributes/unix_sigpipe.
+
+#![cfg_attr(any(sig_dfl, sig_ign), feature(unix_sigpipe))]
+
+#[cfg_attr(sig_dfl, unix_sigpipe = "sig_dfl")]
+#[cfg_attr(sig_ign, unix_sigpipe = "sig_ign")]
```

Comment:
```suggestion
#[cfg_attr(sig_ign, unix_sigpipe = "sig_ign")]
#[cfg_attr(inherit, unix_sigpipe = "inherit")]
```
Isn't this missing? I assume the test still works because it happens to be the same as the default.

Reply:
Good catch, thanks! It indeed passed by accident. I extended the test in a way that would have caught this mistake.